dlhub_cli/commands/describe.py

```diff
@@ -58,32 +58,29 @@ def _preprocess_metadata(metadata):
            You can optionally specify both owner and servable name as a single argument using
            a "owner_name/servable_name" format
            """)
-@click.argument('owner', default=None)
 @click.argument('name', default="")
```

Comment:
Instead of doing the "Model Name Missing" check manually, you can change the default of name to "None." That will use click's automatic "argument missing" logic. Could you change the default and remove the "name == """ test? [... and verify I'm correct about click]